### DISCUSSION TITLE
my account is blocked. pls help

### DISCUSSION THREAD

**Discussion by LSYbuilds** [vercel/vercel#10457] [Help]
### Question

![image](https://github.com/vercel/vercel/assets/130975837/95571875-6054-4ced-9dce-f4d1247b9357)

This is my first time signing up on vercel with my github account but I'm getting an error that say: This user account is blocked. Contact [registration@vercel.com](mailto:registration@vercel.com) for more information. I tried searching for where to contact your member but can't find the link. Pls help me out, I need a solution.

**Comment by MonkeyInWind0** [comment]
+1
![image](https://github.com/vercel/vercel/assets/34543887/121cb5c0-d2ed-483a-a0fb-485323316718)
